<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>back_test_dialog</class>
 <widget class="QDialog" name="back_test_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="btProgressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>441</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QDateEdit" name="btStartDate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>테스트 시작일</string>
   </property>
  </widget>
  <widget class="QComboBox" name="timeCombo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>76</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true">5m</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15m</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1h</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4h</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>단위시간</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btStartBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>테스트 시작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCloseBtn">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>342</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
